src/components/AvatarWithImagePicker.tsx

```diff
@@ -283,7 +283,7 @@ function AvatarWithImagePicker({
 
     return (
         <View style={StyleSheet.flatten([styles.alignItemsCenter, style])}>
-            <View style={[styles.pRelative, avatarStyle]}>
+            <View style={errors && styles.w100}>
```

Comment:
This won't work for the user profile if an error is set:

![image](https://github.com/Expensify/App/assets/27287420/5c17277d-76cc-49b2-921b-e5885d42c055)